diagnostic_feedback/templates/studio/wizard.html

```diff
@@ -0,0 +1,90 @@
+<div class="diagnostic-feedback diagnostic-feedback-studio">
+    <div class="editor">
+        <div class="wrapper-comp-settings is-active editor	">
+            <h1 class="wizard-loading">Loading Wizard, Please wait</h1>
+            <div id="edit_questionnaire_panel" style="display: none;">
+                <div class="msg"><h3></h3></div>
+                <form id="questionnaire-form" action="#">
+                    <div>
+                        <!-- Step 1 start -->
+                        <h3>Basic Info</h3>
+                        <section step="1">
+                        <hr/>
+                            <ul class="list-input settings-list">
+                                <li class="field comp-setting-entry metadata_entry">
+                                    <div class="wrapper-comp-setting">
+                                        <label for="title" class="label setting-label">Quiz Title</label>
+                                        <input id="title" name="title" type="text" class="required field-data-control" value="{{self.title}}">
+                                    </div>
+                                </li>
+                                <li class="field comp-setting-entry metadata_entry">
+                                    <div class="wrapper-comp-setting">
+                                    {% if not self.quiz_type %}
+                                        <label for="type" class="label setting-label">Quiz Type</label>
+                                        <select id="type" name="type" class="required field-data-control">
+                                            <option value=""> -- Select Type -- </option>
+                                            {% for t in self.types %}
+                                                <option value="{{t.value}}" {% if t.value == self.quiz_type %} selected="selected" {% endif %} >
+                                                    {{t.label}}
+                                                </option>
+                                            {% endfor %}
+                                        </select>
+                                    {% else %}
+                                            <label for="type" class="label setting-label">Quiz Type</label>
+                                            <select id="type" name="type" class="required field-data-control" disabled="disabled">
```

Comment:
Do you really need to duplicate all this HTML? Why not just use the same HTML as above but with a disabled attribute, or better yet (since a disabled `<select` is hard to distinguish in the edX color scheme on some platforms), just display `<p>Quiz type: {{ self.quiz_type.label }} (cannot be changed)</p>`